src/operator/join_range.rs

```diff
@@ -68,7 +68,7 @@ where
     ///
     /// This operator is non-incremental, i.e., it joins the pair of batches it
     /// receives at each timestamp ignoring previous inputs.
-    pub fn stream_join_range_indexed<RF, JF, It, K, V, I2>(
+    pub fn stream_join_range_index<RF, JF, It, K, V, I2>(
```

Comment:
which one is the verb? "join" or "index"?
if both perhaps it should be join_and_index
maybe "indexed" is actually ok.

Reply:
Maybe we should standardize on the `_and_index` suffix. It's not uncommon in Rust.